ui/src/app/applications/components/application-status-panel/application-status-panel.scss

```diff
@@ -74,9 +88,42 @@
             top: -2px;
         }
     }
-    &__item-name, &__item-value {
+    &__item-name,
+    &__item-value {
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
     }
+
+    & .info-popup-title {
```

Comment:
CSS styles started from this line were deleted in this commit: https://github.com/argoproj/argo-cd/commit/7165b45aeaa15044cec514e7de3d300d448b2df9#diff-4b88f846774c449afe371a91f0a5184f764cc1cc98a947003281c74407dc4074L82

Please delete obsolte styles